pysat/_instrument.py

```diff
@@ -236,7 +236,17 @@ def __init__(self, platform=None, name=None, tag=None, sat_id=None,
                 raise ValueError(estr)
 
         # set up empty data and metadata
-        self.data = DataFrame(None)
+        # check if pandas or xarray format
+        if self.pandas_format:
+            self._null_data = DataFrame(None)
+            self._data_library = DataFrame
+        else:
+            import xarray
```

Comment:
Earlier you imported xarray as xr.  It's probably a good idea to use the same nicknames throughout.

Reply:
You are correct. When I first started the xarray transition I was hiding the import xarray statements in areas where only the xarray using folks would run into it. Halfway through I decided that really didn't make sense and I should just include xarray as a requirement. Thanks for the catch.